backend/telemetry/src/main/java/org/sonarsource/sonarlint/core/telemetry/measures/payload/TelemetryMeasuresBuilder.java

```diff
@@ -85,6 +85,7 @@ private void addConnectedModeMeasures(ArrayList<TelemetryMeasuresValue> values)
       values.add(new TelemetryMeasuresValue("shared_connected_mode.imported", String.valueOf(storage.getImportedAddedBindingsCount()), INTEGER, DAILY));
       values.add(new TelemetryMeasuresValue("shared_connected_mode.auto", String.valueOf(storage.getAutoAddedBindingsCount()), INTEGER, DAILY));
       values.add(new TelemetryMeasuresValue("shared_connected_mode.exported", String.valueOf(storage.getExportedConnectedModeCount()), INTEGER, DAILY));
+      values.add(new TelemetryMeasuresValue("shared_connected_mode.remote_url", String.valueOf(storage.getSuggestedRemoteBindingsCount()), INTEGER, DAILY));
```

Comment:
I don't see a test to verify that the measure is sent, you could check in the codebase for other measures but I'm pretty sure there will be tests around them